pulled new Thrift definition from Apache Hive repo

The-Alchemist opened the pull request:
Pulled Thrift def from https://raw.githubusercontent.com/apache/hive/master/service-rpc/if/TCLIService.thrift

This version includes `TIMESTAMPLOCALTZ_TYPE`, which was causing problems for me.  See https://github.com/apache/incubator-superset/issues/7520

The-Alchemist commented:
Fixes #285

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/dropbox/PyHive?pullRequest=292) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/dropbox/PyHive?pullRequest=292) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/dropbox/PyHive?pullRequest=292) it.</sub>

jphalip commented:
Thank you for working on this patch. What is the current status? What would it take to merge it? Thanks!

The-Alchemist commented:
> Thank you for working on this patch. What is the current status? What would it take to merge it? Thanks!

I haven't looked at this issue for a few years now. :)

Looks like just a small merge conflict in a unit test.

jphalip commented:
Right, yeah it's been a few years now :)

I too ran into the issue with `TIMESTAMPLOCALTZ`. By chance are you in a position to update the PR, or would you need some help? If the latter, let me know if you have any tips on how I can pick it up. Thanks!

The-Alchemist commented:
> I too ran into the issue with `TIMESTAMPLOCALTZ`. By chance are you in a position to update the PR, or would you need some help? If the latter, let me know if you have any tips on how I can pick it up. Thanks!

@jphalip : I rebased it and fixed all the issues, I believe.  Note, though, I did not regenerate from the latest Thrift definition, but it appears there's no significant changes.

jphalip commented:
@The-Alchemist Thanks a lot! I appreciate it. I can look into updating to the latest definition.

Is it just a matter of running the following?

```sh
python generate.py https://raw.githubusercontent.com/apache/hive/master/service-rpc/if/TCLIService.thrift
```

The-Alchemist commented:
> @The-Alchemist Thanks a lot! I appreciate it. I can look into updating to the latest definition.
> 
> Is it just a matter of running the following?
> 
> ```shell
> python generate.py https://raw.githubusercontent.com/apache/hive/master/service-rpc/if/TCLIService.thrift
> ```

I think so, but it's been a while.  Good luck =)

jphalip commented:
I've just submitted an alternative PR here: #473
